students/1st/feature:1st:kimjihyun/js/login.js

```diff
@@ -0,0 +1,33 @@
+const pressId = document.querySelector('#loginID');
+const pressPW = document.querySelector('#loginPW');
+const button = document.querySelector('.bucksBtn');
+
+// 아이디와 패스워드의 입력 값 체크 함수
+function loginLengthCheck () {
+    const idLength = pressId.value.length;
+    const pwLength = pressPW.value.length;
+
+    button.disabled = !(idLength && pwLength);
+}
+
+// 아이디와 패스워드의 특수 문자 및 입력 값의 길이 체크 함수
+function inputIdCheck () {
+    if (pressId.value.includes('@')) {
+        pressId.classList.add('active');
+    } else {
+        pressId.classList.remove('active');
+    }
+    loginLengthCheck();
+}
+
+function inputPWCheck () {
+    if (pressPW.value.length >= 8) {
+        pressPW.classList.add('active');
```

Comment:
8과 같은 고정값은
`NAMES_LIKE_THIS`, `MIN_PASSWORD_LENGTH`
와 같은 네이밍 컨벤션으로 따로 변수로 관리해주세요.
```suggestion
function inputPWCheck () {
    const MIN_PASSWORD_LENGTH = 8;
    if (pressPW.value.length >= MIN_PASSWORD_LENGTH) {  // 코드를 문장 읽듯이 읽을 수 있습니다.
        pressPW.classList.add('active');
```

Reply:
정말 좋은 방법 이네요.. 미리 숫자 값을 변수로 선언해준 뒤 관리하는 부분은 전혀 생각지도 못했어요..! 
감사합니다 장현님 참고해서 수정하겠습니다. 😄